demo.js

```diff
@@ -0,0 +1,3 @@
+if(true){
```

Comment:
Let's have a space between if and condition
```suggestion
if (true) {
```